fusor/fusor.py

```diff
@@ -549,12 +551,16 @@ def _location_descriptor(
             Descriptor's id. `False`, use default of `label` > `sequence_id`
         """
         seq_id_input = sequence_id
+
         try:
             sequence_id = coerce_namespace(sequence_id)
         except ValueError:
-            try:
-                CURIE(__root__=sequence_id)
-            except ValidationError:
+            # following no longer throws validation error after v2 change??
+            # try:
+            # CURIE(sequence_id)
+            # except ValidationError:
+            #     sequence_id = f"sequence.id:{sequence_id}"
+            if not re.match(r"^\w[^:]*:.+$", sequence_id):
```

Comment:
I would do so you're not hard coding. Then remove the above commented out code

```suggestion
            if not re.match(CURIE.__metadata__[0].pattern, sequence_id):
```

Reply:
this is a huge help, thank you!